CRM/CiviAwards/Setup/AddAwardsCgExtendsOptionValue.php

```diff
@@ -33,4 +36,28 @@ public function apply() {
     ]);
   }
 
+  /**
+   * Enables/Disables the Awards option values
+   *
+   * @param $newStatus
+   *   True to enable, False to disable.
+   */
+  public function toggle($newStatus) {
```

Comment:
This function name needs to be a bit more descriptive. I would rename it to toggleStatus.
Also you need to explain in the description why toggling the status is not enough and why you also need to update the description. 
Normally setting the option value status to FALSE should suffice but there is a bug in Core, I put up am PR for it here (https://github.com/civicrm/civicrm-core/pull/15759) and when this is merged, we would not need to update the description field anymore.